app/models/aliquot.rb

```diff
@@ -18,7 +18,7 @@ class Aliquot < ApplicationRecord
   # maybe we need to do this when the state is set to used?
   # we also need to consider that aliquot can be attached to a sample
   # which will not have a barcode or insert size
-  validates :volume, :concentration, :template_prep_kit_box_barcode, :insert_size, presence: true
+  # validates :volume, :concentration, :template_prep_kit_box_barcode, :insert_size, presence: true
```

Comment:
Is this the best way to do this?

Reply:
Yeah I did wonder if we want to only validate them if source is not on a request until (if) we add request volume tracking. Would that be better?